options.css

```diff
@@ -0,0 +1,82 @@
+#filter-footer {
+    display: flex;
+    width: 100%;
+    align-items: center;
+    justify-content: center;
+}
+
+#submit-button {
+    letter-spacing: 0.14em;
+    text-transform: uppercase;
+    font-size: 16px;
+    line-height: 16px;
+    margin-top: 24px;
+    outline: none;
+    padding: 10px 24px;
+    border: 0;
+    border-radius: 4px;
+    background-color: #f3f3f3;
+    color: #767676;
+    text-align: center;
+    cursor: pointer;
+}
+
+#submit-button:disabled {
+    opacity: 0.6;
+    cursor: not-allowed;
+}
+
+#filter-status {
+    font-size: 14px;
+    font-family: "Ideal Sans A", "Ideal Sans B", Helvetica, Arial, sans-serif;
```

Comment:
Although we use Ideal Sans on the website, we can't use it here for licensing reasons. Let's use Open Sans instead. Our designer recommended it as the alternative to Ideal Sans for the context. We already use it for the new tab itself:

https://github.com/art-institute-of-chicago/browser-extension/blob/bc41582ed38398423aa9ecc564230b365f46386a/index.html#L6

We will for sure check in with her about the options page once we've got more filters implemented, but for now, let's use Amiri for headlines and Open Sans for everything else.